src/env.ts

```diff
@@ -0,0 +1,16 @@
+const SECRET_KEY_MINIMUM_LENGTH = 44;
+
+// biome-ignore lint/complexity/useLiteralKeys: tsc complains about this (TS4111)
+const secretKey = process.env["SECRET_KEY"];
+
+if (typeof secretKey !== "string") {
+  throw new Error("SECRET_KEY is required");
+}
+
+if (secretKey.length < SECRET_KEY_MINIMUM_LENGTH) {
+  throw new Error(
+    `SECRET_KEY is too short, received: ${secretKey.length}, expected: ${SECRET_KEY_MINIMUM_LENGTH}`,
+  );
```

Comment:
Could we make it just a warning and let just go this time? We could require it in the future.

Also, could you mention this in *docs/src/content/docs/install/env.mdx*? I'll translate it later then.

Reply:
I wouldn't, as an insecure secret key value is security problem. In other software, it'll use the SECRET_KEY as a base from which to derive more keys using like pbkdf2 or something, which ensures sufficient key material.

I'm not sure exactly how subtle crypto would handle with a single character key for instance? Would it throw?

Reply:
Hmm, fair enough. Out of curiosity, is there a special meaning to the number 44?

Reply:
That's the shorter length given by the provided openssl commands, so hopefully it won't break anyone's install.

Reply:
I got it!